phoenix-hive-base/pom.xml

```diff
@@ -29,42 +29,46 @@
         <artifactId>phoenix-connectors</artifactId>
         <version>6.0.0-SNAPSHOT</version>
     </parent>
-    <artifactId>phoenix-hive3</artifactId>
-    <name>Phoenix Hive Connector for Phoenix 5</name>
+    <artifactId>phoenix-hive-base</artifactId>
+    <name>Phoenix Hive Connector - Base</name>
+
+    <packaging>pom</packaging>
+    <modules>
+        <module>phoenix4-hive</module>
+        <module>phoenix5-hive</module>
+    </modules>
+
     <properties>
         <test.tmp.dir>${project.build.directory}/tmp</test.tmp.dir>
-        <netty.version>4.1.47.Final</netty.version>
-        <phoenix.version>5.1.0-SNAPSHOT</phoenix.version>
-        <hbase.version>2.2.4</hbase.version>
-        <hadoop.version>3.0.3</hadoop.version>
-        <avatica.version>1.12.0</avatica.version>
-        <hive3.version>3.1.2</hive3.version>
-        <curator.version>4.0.0</curator.version>
+        <netty.version>4.0.52.Final</netty.version>
```

Comment:
These dependencies are pretty ancient.
Use the hive 3 dependency versions, if you can.